packages/conf-libev/conf-libev.4-11/opam

```diff
@@ -9,4 +9,8 @@ depexts: [
   [["fedora"] ["libev-devel"]]
   [["rhel"] ["libev-devel"]]
   [["centos"] ["libev-devel"]]
+  [["opensuse"] ["libev-devel"]]
+  [["freebsd"] ["libev"]]
+  [["openbsd"] ["devel/libev"]]
```

Comment:
the openbsd one should just be `libev`, as it will get added via `pkg_add` which operates on binary packages (and not the ports tree)